invenio_records_resources/services/errors.py

```diff
@@ -69,3 +69,12 @@ def __init__(self, recid, file_key):
         )
         self.recid = recid
         self.file_key = file_key
+
+
+class RecordDeletedException(Exception):
```

Comment:
this was moved here from invenio-rdm-records as it's required in this module as well